chapters/glossary.tex

```diff
@@ -81,11 +84,7 @@ \chapter{Glossary}\doublelabel{glossary}
   components and classes. Example reference: \lstinline!Ele.Resistor.u[21].r! (See
 \autoref{component-declarations} and \autoref{slice-operation}.)
 
-\textbf{declaration assignment}: assignment of the form \lstinline!x := expression!
-defined by a component declaration. This is similar to a declaration
-equation. In contrast to a declaration equation, a declaration
-assignment is allowed only when declaring a component contained in a
-function. (See \autoref{initialization-and-declaration-assignments-of-components-in-functions}.)
+\textbf{declaration assignment}: Deprecated name for binding equation in function.
```

Comment:
Would be good to know which version it was deprecated.

Reply:
The name will be deprecated in 3.5 (this PR), but the assignments themselves were deprecated in the standard 3.4 (#1595) - based on existing practice.
I'm not sure if adding that in the glossary will be helpful.

Reply:
OK. No problem.